HelmetCamera/HelmetCamera.cs

```diff
@@ -112,29 +123,38 @@ private IEnumerator LoadSceneEnter()
-            bool isCameraFound = GameObject.Find("Environment/HangarShip/Cameras/ShipCamera") != null;
+
+            // Reduce amount of searches by caching
+            GameObject foundHangarShip = GameObject.Find("Environment/HangarShip");
+            Transform foundMonitorWall = foundHangarShip.transform.Find("ShipModels2b/MonitorWall");
+            Transform foundShipCamera = foundHangarShip.transform.Find("Cameras/ShipCamera");
+
+            bool isCameraFound = foundShipCamera != null;
             if (isCameraFound)
             {
-                Debug.Log("[HELMET_CAMERAS] Ship camera founded...");
+                Debug.Log("[HELMET_CAMERAS] Ship camera found...");
                 if (!isMonitorChanged)
                 {
-                    GameObject.Find("Environment/HangarShip/ShipModels2b/MonitorWall/Cube")
-                        .GetComponent<MeshRenderer>().materials[2].mainTexture = GameObject.Find("Environment/HangarShip/ShipModels2b/MonitorWall/Cube.001")
+                    Transform foundCube001 = foundMonitorWall.Find("Cube.001");
+                    
+                    foundMonitorWall.Find("Cube")
+                        .GetComponent<MeshRenderer>().materials[2].mainTexture = foundCube001
                         .GetComponent<MeshRenderer>().materials[2].mainTexture;
-                    GameObject.Find("Environment/HangarShip/ShipModels2b/MonitorWall/Cube.001").GetComponent<MeshRenderer>().materials[2].mainTexture = renderTexture;
-
-                    helmetCameraNew.AddComponent<Camera>();
-                    helmetCameraNew.GetComponent<Camera>().enabled = false;
-                    helmetCameraNew.GetComponent<Camera>().targetTexture = renderTexture;
-                    helmetCameraNew.GetComponent<Camera>().cullingMask = 20649983;
-                    helmetCameraNew.GetComponent<Camera>().farClipPlane = renderDistance;
-                    helmetCameraNew.GetComponent<Camera>().nearClipPlane = 0.55f;
+                    foundCube001.GetComponent<MeshRenderer>().materials[2].mainTexture = renderTexture;
+                    
+                    // Reduce get calls by using reference to camera
+                    Camera newCamera = helmetCameraNew.AddComponent<Camera>();
+                    newCamera.enabled = false;
+                    newCamera.targetTexture = renderTexture;
+                    newCamera.cullingMask = 20649983;
+                    newCamera.farClipPlane = renderDistance;
+                    newCamera.nearClipPlane = 0.55f;
 
                     isMonitorChanged = true;
                     Debug.Log("[HELMET_CAMERAS] Monitors were changed...");
 
                     Debug.Log("[HELMET_CAMERAS] Turning off vanilla internal ship camera");
-                    GameObject.Find("Environment/HangarShip/Cameras/ShipCamera").GetComponent<Camera>().enabled = false;
+                    foundShipCamera.GetComponent<Camera>().enabled = false;

```

Comment:
I think this might not actually disable the camera, since ManualCameraRenderer will be enabling the internal ship camera.

Not only that, but disabling it successfully may cause the map in the terminal to stop functioning, see the comments in [this patch from OpenBodyCams](https://github.com/Zaggy1024/LC_OpenBodyCams/blob/254555a1d8941647f4c1160174b0aa6d01bc4995/OpenBodyCams/Patches/PatchManualCameraRenderer.cs#L42-L94).